src/status_im/navigation/state.cljs

```diff
@@ -2,6 +2,36 @@
 
 (defonce root-id (atom nil))
 (defonce pushed-screen-id (atom nil))
-(defonce curr-modal (atom nil))
 (defonce modals (atom []))
 (defonce dissmissing (atom false))
```

Comment:
dissmissing is very vague. What exactly is this used for and can it be more descriptive. Should it also be `dismissing?` since it seems it's a boolean?

Reply:
Actually its not **only** boolean - https://github.com/status-im/status-mobile/blob/aa7db8b883823ced53481f1f5cad461b2b0b0df4/src/status_im/navigation/effects.cljs#L129

> What exactly is this used for 

For dismissing modals sequentially

> dissmissing is very vague

We can explore refactoring more in follow up PRs.

Reply:
so it's more like a queue of components to be dismissed?
should it not be `nil` by default?